ctapipe/image/muon/tests/test_features.py

```diff
@@ -1,5 +1,10 @@
 import astropy.units as u
+import numpy as np
+import math as mt
```

Comment:
is `import math as mt` common in the community? These `as something` imports effectively introduce new "global variables". And we all know globalb variables are bad ... but when a python-scientists reads `np.something` she just *knows* that np is numpy. So it is fine. 

Same goes for `plt`, `pd` and `u`... `mt` is new for me.

Reply:
Unfortunately I have never found a list online, which lists these canonical module aliases ... So I don't know if `mt` is canonical.

Reply:
Not sure if `mt` is canonical, I reverted it to `math`